### ISSUE TITLE
DL3041: False positives when package version contains a colon

### ISSUE THREAD

**Issue by carlsmedstad** [hadolint/hadolint#1117]
Version 2.13.1 introduced a false positive in the check DL3041 (_Specify version with `dnf install -y <package>-<version>`_) when a version contains a colon, used to declare [package epoch](https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_epoch_tag) for RPM packages.

Example:

```dockerfile
FROM registry.access.redhat.com/ubi8/ubi-minimal:8.10

# DL3041 triggerd by this:
RUN dnf install --assumeyes \
    jq-1.6 \
    openssl-1:1.1.1k \
    findutils-1:4.6.0 \
    bc-1.07.1 \
    && dnf clean all
```

Removing the colons from the above also removes the DL3041 warning:

```dockerfile
FROM registry.access.redhat.com/ubi8/ubi-minimal:8.10

RUN dnf install --assumeyes \
    jq-1.6 \
    openssl-1.1.1k \
    findutils-4.6.0 \
    bc-1.07.1 \
    && dnf clean all
```

**Comment by m-ildefons** [comment]
Release v2.14.0 is out and includes the linked PR with the fix. I'll close this as completed.
